packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -224,41 +240,54 @@ async function checkDependencies(
 async function resolveBackendExtension(
   depsManager: DepsManager,
   projectSettings: ProjectSettings
-): Promise<CheckFailure | undefined> {
+): Promise<CheckResult> {
+  let result = true;
+  let error = undefined;
   try {
     if (ProjectSettingsHelper.includeBackend(projectSettings)) {
       const backendRoot = path.join(ext.workspaceUri.fsPath, FolderName.Function);
       const dotnet = (await depsManager.getStatus([DepsType.Dotnet]))[0];
-      // TODO: check before install backend extension
-      await installExtension(backendRoot, dotnet.command, vscodeLogger);
+      await installExtension(backendRoot, dotnet.command, doctorLogger);
     }
-    return undefined;
-  } catch (error: any) {
-    return {
-      checker: "Backend Extension",
-      error: handleDepsCheckerError(error),
-    };
+  } catch (err: any) {
+    result = false;
+    error = handleDepsCheckerError(err);
   }
+  return {
+    checker: "Backend Extension",
+    result: result,
+    error: error,
+  };
 }
 
-async function resolveLocalCertificate(
-  localEnvManager: LocalEnvManager
-): Promise<CheckFailure | undefined> {
+async function resolveLocalCertificate(localEnvManager: LocalEnvManager): Promise<CheckResult> {
+  let result = true;
+  let error = undefined;
   try {
     const trustDevCert = vscodeHelper.isTrustDevCertEnabled();
 
-    // TODO: Return CheckFailure when isTrusted === false
+    // TODO: Return CheckResult when isTrusted === false
     await localEnvManager.resolveLocalCertificate(trustDevCert);
-    return undefined;
-  } catch (error: any) {
-    return {
-      checker: "Local Certificate",
-      error: assembleError(error),
-    };
+  } catch (err: any) {
+    result = true;
+    error = assembleError(err);
```

Comment:
If trust certificate failes but debug continues, can we use a warning instead of success?
[!] Local Certification
error message ......

Or we can stop the debugger when trust cert failed.

Reply:
This is my mistake, I didn't intend to change the logic here. we should stop the debugger like before.

Reply:
Fixed.